docs/en/sql-reference/table-functions/view.md

```diff
@@ -0,0 +1,67 @@
+---
+toc_priority: 51
+toc_title: view
+---
+
+## view {#view}
+
+Turns a subquery into a table. The function implements views (for more information, see the [CREATE VIEW](https://clickhouse.tech/docs/en/sql-reference/statements/create/view/#create-view). The resulting table doesn't store data, but only stores the specified `SELECT` query. When reading from the table, ClickHouse executes the query and deletes all unnecessary columns from the result.
```

Comment:
- "the CREATE VIEW query" or "CREATE VIEW"
- "For more information" is a redundant phrase with zero use. Please delete it.